_editorial_policies/02_editor_instructions.md

```diff
@@ -56,10 +56,24 @@ Once an editor has handled the pre-review steps described above, the review proc
 
 Following acceptance, the procedure is:
 - Make sure the metadata is complete in the submission (including institution, ORCID, abstract). 
-- Have the managing editor assign a DOI to the article (correct metadata is needed for this).
-- Have the author do a final check for issues with grammar, word choice, spelling, formatting (including issues with the LaTeX template, placement of figures, etc.) and typos, as normally would be done at the proofs stage. The editor should provide some light assistance if it may be helpful to getting the article out in a timely manner.
-- Have the author enable the LiveCoMS footer by including the 'pubversion' class option in the document preamble, fill in the '\datereceived', '\dateaccepted', and '\pubDOI' fields, and provide a recompiled PDF for posting to the editor handling the submission and a managing editor.  The PDF should be accessible on the GitHub repository as `release/LiveCoMS_Article_VX.pdf`, where 'X' is the current article verrsion. The related header image should be at least 1200 pixels wide and in `.jpg` format, and should be available on the GitHub repository as `release/header_VX.jpg`, where 'X' is the current article version.
-- Have the author provide a "lede", a 30 word summary of the article that will accompany the article in social media posts.
-- Have a managing editor post the PDF as an ASAP article.
-- When assigned to an issue, fill in (or have the author fill in) the '\pubvolume', '\pubyear', and '\articlenum' fields, recompile the PDF, and then provide to a managing editor to update the PDF.
+- Email the managing editors (managing@livecomsjournal.org) to obtain a DOI for the article (correct metadata is needed for this); they will take the metadata directly from the Scholastica website.
```

Comment:
* "AUTHORS to email the managing editors ... ; MANAGING EDITORS will take ..." [is this really two separate steps? should we split it up?]

Reply:
I'm not clear what you mean here.  The authors shouldn't email the managing editors, the editors should.  I've tried to clarify that in an update.